deploy/helm/grafana-operator/README.md

```diff
@@ -14,11 +14,23 @@ linkTitle: "Helm installation"
 This is a OCI helm chart, helm started support OCI in version 3.8.0.
 
 ```shell
-helm upgrade -i grafana-operator oci://ghcr.io/grafana/helm-charts/grafana-operator --version v5.7.0
+helm upgrade -i grafana-operator oci://ghcr.io/grafana/helm-charts/grafana-operator --version v5.8.1
 ```
 
 Sadly helm OCI charts currently don't support searching for available versions of a helm [oci registry](https://github.com/helm/helm/issues/11000).
 
+## Upgrading
+
+Helm does not provide functionality to update custom resource definitions. This can result in the operator misbehaving when a release contains updates to the custom resource definitions.
+To avoid issues due to outdated or missing definitions, run the following command before updating an existing installation:
+
+```shell
+kubectl apply --server-side --force-conflicts -f https://github.com/grafana/grafana-operator/releases/download/v5.8.1/crds.yaml
```

Comment:
Should we write something about, adjust the release version to the helm chart version. Or add some variable that we use for both the commands?

Reply:
Variables would be nice, but I'm not sure how frequently people check out the doc on the website (where variables work) vs viewing the README on GitHub (where they don't) :thinking:. What's your take on this?

Reply:
Right now, we are using variables everywhere in the document. I see your point of that many people just check directly in the helm chart. But at the same time, it would be nice if we have some kind of automation around this for the documentation page as well.

So if we should change this, I think we should do it in a separate PR.